localisation/00_Empire_names.csv

```diff
@@ -25,8 +25,8 @@ e_kul_tiras;Kul Tiras;Kul Tiras;Kul Tiras;;Kul Tiras;;;;;;;;;x
 e_kul_tiras_adj;Kul Tirasian;;Kul Tiraser;;;;;;;;;;;x
 e_lordaeron;Lordaeron;Lordaeron;Lordaeron;;Lordaeron;;;;;;;;;x
 e_lordaeron_adj;Lordaeronian;;lordaeroner;;;;;;;;;;;x
-e_quelthalas;Quel'thalas;Quel'thalas;Quel'thalas;;Quel'thalas;;;;;;;;;x
-e_quelthalas_adj;Quel'thalasian;;quel'thalaser;;;;;;;;;;;x
+e_quelthalas;Quel'Thalas;Quel'Thalas;Quel'Thalas;;Quel'Thalas;;;;;;;;;x
+e_quelthalas_adj;Quel'Thalasian;;quel'thalaser;;;;;;;;;;;x
 e_storm_peaks;Storm Peaks;Storm Peaks;Storm Peaks;;Storm Peaks;;;;;;;;;x
 e_storm_peaks_adj;Storm Peaksian;;Storm Peakser;;;;;;;;;;;x
```

Comment:
I will admit that I have no better ideas, but is Peaksian really the option?

Reply:
Maybe not best, but keeping the style... It's worse for non-english localisation...

Reply:
Stormic?

Reply:
I would change to just **Storm**. Though, there's a [Problem](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/wiki/Rules#localization-rules).

Reply:
I'd go with Zumbak's version. CK2 will do the rest